scripts/conflicts/diff.js

```diff
@@ -10,8 +11,13 @@ const argv = parseArgs(process.argv);
 
 const server = argv._[2] || argv.server || process.env.COUCH_URL;
 
+// Example
+//server = 'https://user:pass@medic-bhaktapur.com/medic'
```

Comment:
Please change this example to show the way to actually pass the server url as an environment variable or argument, instead of instructing on how to hardcode it.